docs/prd.md

```
@@ -387,36 +389,36 @@ The EAF itself (as an admin/operator portal) will require:
 
 ---
 
-## Epic 8: Licensing Server (MVP Validation)
+## Epic 9: Licensing Server (MVP Validation)
 
-**Epic Goal:** This is the capstone epic for the v0.1 MVP. Its purpose is not to build a new core feature, but to **validate the entire framework** by being the first *product* built using only the EAF components from Epics 1-7. This epic directly fulfills all three MVP Success Criteria.
+**Epic Goal:** This is the capstone epic for the v0.1 MVP. Its purpose is not to build a new core feature, but to **validate the entire framework** by being the first *product* built using only the EAF components from Epics 1-8. This epic directly fulfills all three MVP Success Criteria.
 
-#### Story 8.1: Scaffold the 'Licensing-Server' Product Module
+#### Story 9.1: Scaffold the 'Licensing-Server' Product Module
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)